### PR TITLE
Add Creditum

### PR DESCRIPTION
**Pull request by shatterproof**
**New Tokens:**
- CREDIT
- xCREDIT
- cUSD

**New Pools:**
- Curve cUSD-USDC-DAI
- Spooky CREDIT-FTM
- 0xDAO xCREDIT

**Additional Improvements:**
- Optimized multiple Curve APY calculations by using `getCurveFactoryApy`
- Added logo URI for binSPIRIT

### CODE REVIEW

File: src/api/stats/fantom/getCreditumApys.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    const totalApy = getFarmWithTradingFeesApy(
+      simpleApy,
+      tradingApr,
+      BASE_HPY,
+      1,
+      shareAfterBeefyPerformanceFee
+    );
+
+    // Create reference for legacy /apy
+    const legacyApyValue = { [pool.name]: totalApy };
```
Comment (roman-monk) [inline src/api/stats/fantom/getCreditumApys.js:87]: there is `getApyBreakdown()` which should simplify this breakdown building

Comment (roman-monk) [reply src/api/stats/fantom/getCreditumApys.js:87]: mmm, totalAllocPoints doesnt match, ok

File: src/api/stats/fantom/getCreditumApys.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+import { compound } from '../../../utils/compound';
+
+const masterchef = '0xe0c43105235C1f18EA15fdb60Bb6d54814299938';
+const rewardToken = 'CREDIT';
+const rewardDecimals = '1e18';
+const secondsPerBlock = 1;
+const secondsPerYear = 31536000;
+const curvePool = '0x96059756980fF6ced0473898d26F0EF828a59820';
+
+const getCreditumApys = async () => {
```
[the remark this answers is not in the document]
Comment (shatterproof) [reply src/api/stats/fantom/getCreditumApys.js:29]: Tried that originally but it didn't seem to fit due to the different format of the SteakHouse masterchef it uses (there is no `tokensPerBlock` string to use unlike getSpellApys)

I can look deeper into it tomorrow now that I am more familiar with the code, if it works I could also simplify getSteakHouseLpApys in the same way

[the remark this answers is not in the document]
Comment (roman-monk) [reply src/api/stats/fantom/getCreditumApys.js:29]: tokensPerBlock is customizable:
```
tokenPerBlock: "RewardsPerSecond"
secondsPerBlock: 1
```
allocPointIndex can be changed too.

but yeah i see that both tokenPerBlock and totalPoints require index, no need to adjust it then, lets keep as is

**Pull request merged**